www/syncStatus.ts

```diff
@@ -0,0 +1,32 @@
+/// <reference path="../typings/codePush.d.ts" />
+
+"use strict";
+
+/**
+ * Defines the possible result statuses of the window.codePush.sync operation.
+ */
+enum SyncStatus {
```

Comment:
Would it make sense to put this enum in the common SDK? There's a lot of overlap here. Although, there is a small inconsistency in that the Cordova implementation exposes an ERROR enum, whereas the RN implementation rejects the promise.